src/main/java/ru/ayaz/User.java

```diff
@@ -0,0 +1,54 @@
+package ru.ayaz;
+
+import java.io.PrintWriter;
+import java.io.Writer;
+import java.security.Principal;
+
+/**
+ * Created by ayaz on 11/5/16.
+ */
+public class User {
+    private String nickName;
+    private long id;
+    private PrintWriter writer;
+
+    public void setNickName(String nickName){
+
+        if((nickName == null)||(nickName.equals(""))){
+            throw new IllegalArgumentException();
+        }
+
+        if(nickName.replaceAll("\\s+", "").equals("")){
```

Comment:
nickName.replaceAll("\\s+", "") or even nickName.replaceAll("\\s+", "").equals("") could be placed inside a method to give it a better explanation.